front/pages/w/[wId]/builder/data-sources/public-urls.tsx

```diff
@@ -271,15 +275,27 @@ function getTableColumns() {
     },
     {
       header: "Used by",
-      accessorKey: "usage",
-      id: "usage",
+      id: "usedBy",
+      accessorKey: "managedDataSource.usage",
```

Comment:
`nit`: you can get rid of the `accessorKey` here, it's useful when using `{info.getValue()}`